yarner-lib/Cargo.toml

```diff
@@ -2,6 +2,12 @@
 name = "yarner-lib"
 version = "0.5.1"
 authors = ["Martin Lange <martin_lange_@gmx.net>"]
+readme = "README.md"
+description = "Library to build plugins for the Literate Programming tool Yarner (https://github.com/mlange-42/yarner)."
```

Comment:
I would recommend not including the URL in the description as this is already part of the repository and homepage links.